tests/e2e/ansible/install_test_deps.yaml

```diff
@@ -39,8 +39,12 @@
       ignore_errors: yes
     - name: Install kustomize
       shell: |
-        curl -s --retry 3 --retry-delay 10 -u ${USER}:${GITHUB_TOKEN} "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
-        cp -f ./kustomize /usr/local/bin
+        version=$(curl -s https://api.github.com/repos/kubernetes-sigs/kustomize/releases | grep '"tag_name":' | grep "kustomize" | head -n 1 | cut -d '"' -f 4 | cut -d "/" -f 2)
+        arch=$(dpkg --print-architecture)
+        tarball="kustomize_${version}_linux_${arch}.tar.gz"
+        curl -Lf -o "$tarball" "https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2F${version}/kustomize_${version}_linux_${arch}.tar.gz"
+        tar -xvzf "${tarball}" -C /usr/local/bin
+        command -v kustomize >/dev/null 2>&1
```

Comment:
Good idea although it changes the behaviour from `install latest master` to `install latest release`. If we are changing the behaviour I'd probably go with pinned known-to-work version rather than potentially unstable query for the latest release. Alternatively we could first install go and then use `GOBIN=$(pwd)/ GO111MODULE=on go install sigs.k8s.io/kustomize/kustomize/v5@latest`

Reply:
Btw the `GOBIN` should be `/usr/local/bin`, the previous command was taken from kustomize documentation ;-)